src/main/java/org/example/enums/ItemType.java

```diff
@@ -0,0 +1,18 @@
+package org.example.enums;
+
+public enum ItemType {
+  RAW,
+  MANUFACTURED,
+  IMPORTED;
+
+  public static ItemType fromString(final String type) {
+    if (type == null) {
```

Comment:
use objects.isnull()

Reply:
used objects.isnull() instead of type == null